addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -290,6 +298,14 @@ var SlideDialog = Widget.extend({
             });
         return res;
     },
+    //Python PIL does not support SVG, so converting SVG to PNG
```

Comment:
PIL library does not support SVG format, and gives traceback "IOError: cannot identify image file" when we upload SVG images(not only when we are using it with slides module, but also from backend for example when we try to set svg image for company logo or partner/user/prodcut's image).